examples/example-gdpr.html

```diff
@@ -4,10 +4,12 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta http-equiv="content-type" content="text/html; charset=utf-8">
     <link href="style.css" rel="stylesheet">
-    <script src="../src/bundle.js"></script>
+    <link href="../build/cookieconsent.min.css" rel="stylesheet">
+    <script src="../build/cookieconsent.min.js"></script>
+    <!-- <script src="../src/bundle.js"></script> --> <!-- for dev testing-->
```

Comment:
What about this? Is this intended change?

Reply:
just for dev testing